.github/workflows/rust.yml

```diff
@@ -160,4 +160,69 @@ jobs:
       - name: cargo test fea-rs
         run: cargo test -p fea-rs
 
+  # Confirm we can still compile GS
+  build-gs:
+    runs-on: ubuntu-latest
+    name: Build GS
+    steps:
+      - name: Check out fontc source repository
+        uses: actions/checkout@v4
+      - name: Install the latest stable Rust toolchain
+        uses: dtolnay/rust-toolchain@stable
+      - name: Build and install fontc
+        run: cd fontc && cargo install --path . --debug
+      - name: Check out font project source repository
+        uses: actions/checkout@v4
+        with:
+          repository: googlefonts/googlesans
+          token: ${{ secrets.GS_READ_FONTC }}
+      - name: Download and unzip OTS v9.1.0
+        run: |
+            curl -OL "https://github.com/khaledhosny/ots/releases/download/v9.1.0/ots-9.1.0-Linux.zip"
+            unzip "ots-9.1.0-Linux.zip" "ots-9.1.0-Linux/ots-sanitize"
+      - name: Compile Roman variable font
+        run: fontc source/GoogleSans/GoogleSans.designspace
+      - name: OTS tests, Roman
+        run: ots-9.1.0-Linux/ots-sanitize build/font.ttf
+      - name: Compile Italic variable font
+        run: fontc source/GoogleSans/GoogleSans-Italic.designspace
+      - name: OTS tests, Italic
+        run: ots-9.1.0-Linux/ots-sanitize build/font.ttf
+
+  # Check if GS build is repeatable, https://github.com/googlefonts/fontc/issues/647
+  build-gs-repeatably:
+    runs-on: ubuntu-latest
+    name: Build GS and get a consistent result
+    steps:
+      - name: Check out fontc source repository
+        uses: actions/checkout@v4
+
+      - name: Install the latest stable Rust toolchain
+        uses: dtolnay/rust-toolchain@stable
 
+      - name: Build and install fontc
+        run: cd fontc && pwd && cargo install --path .
+
+      - name: Check out font project source repository
+        uses: actions/checkout@v4
+        with:
+          repository: googlefonts/googlesans
+          token: ${{ secrets.GS_READ_FONTC }}
+
+      - name: Compile me once, shame on you
+        run: |
+          rm -rf build
+          echo "TEMPORARY DIAGNOSTICS"
+          pwd
+          ls
+          fontc source/GoogleSans/GoogleSans.designspace
+          cp build/font.ttf ./first.ttf
+
+      - name: Compile me twice, shame on me
+        run: |
+          rm -rf build
+          fontc source/GoogleSans/GoogleSans.designspace
+          cp build/font.ttf ./second.ttf
```

Comment:
I suggest you also list the table directory with `ttx -l` for both fonts, or even diff the two directories, 
helpful for inspecting diffs in table sizes/checksums

Reply:
Good idea. I guess that means I have to install python. Hopefully that's fast.

Reply:
I think the ubuntu container may already come with one.. try

Reply:
yeah looks like it already has python https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2204-Readme.md

Reply:
> I think the ubuntu container may already come with one.. try

shiny. Will rig.